### ISSUE TITLE
Support for light theme for WebShell

### ISSUE THREAD

**Issue by notthetup** [org-arl/fjage#197]

**Comment by notthetup** [comment]
https://github.com/xtermjs/xterm.js/blob/4.10.0/typings/xterm.d.ts#L266

**Comment by notthetup** [comment]
From https://github.com/org-arl/fjage/blob/master/src/main/java/org/arl/fjage/shell/ConsoleShell.java

```
  private void setupStyles() {
    AttributedStyle style = new AttributedStyle();
    promptStyle = style.foreground(AttributedStyle.BRIGHT+AttributedStyle.YELLOW);
    inputStyle = style.foreground(AttributedStyle.WHITE);
    outputStyle = style.foreground(AttributedStyle.GREEN);
    notifyStyle = style.foreground(AttributedStyle.BRIGHT+AttributedStyle.BLUE);
    errorStyle = style.foreground(AttributedStyle.RED);
  }
```

**Comment by notthetup** [comment]
So we are using, 

`promptStyle` => `BRIGHT+YELLOW` => 
```
/** ANSI bright yellow (eg. `\x1b[1;33m`) */    
brightYellow?: string;
```

`inputStyle` => `WHITE` => 
``` 
/** ANSI white (eg. `\x1b[37m`) */    
white?: string;
```

`outputStyle` => `GREEN` =>  
``` 
/** ANSI green (eg. `\x1b[32m`) */    
green?: string;
```

`notifyStyle` => `BRIGHT+BLUE` => 
``` 
/** ANSI bright blue (eg. `\x1b[1;34m`) */    
brightBlue?: string;
```

`errorStyle` => `RED` => 
``` 
/** ANSI red (eg. `\x1b[31m`) */    
red?: string;
```

**Comment by notthetup** [comment]
Something like seems to work but needs to be fine tuned.

```
term.setOption('theme', { 
    background: '#fdf6e3',
    yellow: '#002b36',
    foreground: '#839496',
    white: '#839496',
    green: '#859900',
    blue: '#268bd2',
    red: '#dc322f',
    cursor: '#ff0000'
  });
```

**Comment by notthetup** [comment]
Fixed in #198
